packages/shared/components/atoms/buttons/MintNativeTokenButton.svelte

```diff
@@ -1,12 +1,34 @@
 <script lang="typescript">
     import { localize } from '@core/i18n'
-    import { openPopup } from '@lib/popup'
+    import { ActivityType, selectedAccountActivities } from '@core/wallet'
+    import { closePopup, openPopup } from '@lib/popup'
     import { OnboardingButton } from 'shared/components'
+    import { get } from 'svelte/store'
 
-    function handleMintNativeToken() {
-        openPopup({
-            type: 'mintNativeTokenForm',
-        })
+    $: aliases = get(selectedAccountActivities).filter((activity) => activity.type === ActivityType.Alias)
```

Comment:
This should be gotten from wallet.rs (`account.getBalance()`), since you could lose control of an alias in the future